integration/healthcheck_test.go

```diff
@@ -67,16 +55,40 @@ func (s *HealthCheckSuite) TestSimpleConfiguration(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(resp.StatusCode, checker.Equals, 200)
 
-	healthReq, err := http.NewRequest("POST", "http://"+whoami1Host+"/health", bytes.NewBuffer([]byte("500")))
+	// Fix all whoami health to 500
+	whoamiHosts := []string{whoami1Host, whoami2Host}
+	for _, whoami := range whoamiHosts {
+		healthReq, err := http.NewRequest("POST", "http://"+whoami+"/health", bytes.NewBuffer([]byte("500")))
+		c.Assert(err, checker.IsNil)
+		_, err = client.Do(healthReq)
+		c.Assert(err, checker.IsNil)
+	}
+
+	utils.Sleep(3 * time.Second)
```

Comment:
Almost the same here, if we want to wait 3s and then have the `TryRequest`, let's just set a bigger timeout for the `Try`.